src/Compilers/VisualBasic/Test/Semantic/BasicCompilerSemanticTest.vbproj

```diff
@@ -66,149 +65,21 @@
-    <Compile Include="Semantics\NoncompliantOverloadingInMetadata.vb" />
-    <Compile Include="Semantics\OptionalArgumentTests.vb" />
-    <Compile Include="Semantics\OptionApis.vb" />
-    <Compile Include="Semantics\OverloadResolution.vb" />
-    <Compile Include="Semantics\Parenthesized.vb" />
-    <Compile Include="Semantics\PartialMethodsTest.vb" />
-    <Compile Include="Semantics\QueryExpressions.vb" />
-    <Compile Include="Semantics\QueryExpressions_FlowAnalysis.vb" />
-    <Compile Include="Semantics\QueryExpressions_LookupSymbols.vb" />
-    <Compile Include="Semantics\QueryExpressions_SemanticModel.vb" />
-    <Compile Include="Semantics\RedimStatementTests.vb" />
-    <Compile Include="Semantics\ScriptSemanticsTests.vb" />
-    <Compile Include="Semantics\SelectCaseTests.vb" />
-    <Compile Include="Semantics\SyncLockTests.vb" />
-    <Compile Include="Semantics\SyntaxTreeRootTests.vb" />
-    <Compile Include="Semantics\TooLongNameTests.vb" />
-    <Compile Include="Semantics\TypeArgumentInference.vb" />
-    <Compile Include="Semantics\TypeOfTests.vb" />
-    <Compile Include="Semantics\UnaryOperators.vb" />
-    <Compile Include="Semantics\UnstructuredExceptionHandling.vb" />
-    <Compile Include="Semantics\UserDefinedBinaryOperators.vb" />
-    <Compile Include="Semantics\UserDefinedConversions.vb" />
-    <Compile Include="Semantics\UserDefinedForToLoop.vb" />
-    <Compile Include="Semantics\UserDefinedUnaryOperators.vb" />
-    <Compile Include="Semantics\UseSiteErrorTests.vb" />
-    <Compile Include="Semantics\UsingStatementTest.vb" />
-    <Compile Include="Semantics\VariableTypeInference.vb" />
-    <Compile Include="Semantics\VarianceConversions.vb" />
-    <Compile Include="Semantics\WithBlockErrorTests.vb" />
-    <Compile Include="Semantics\WithBlockSemanticModelTests.vb" />
-    <Compile Include="Semantics\XmlLiteralSemanticModelTests.vb" />
     <Content Include="Semantics\Async_Overload_Change_3.vb.txt" />
     <Content Include="Semantics\BinaryOperatorsTestBaseline1.txt" />
     <Content Include="Semantics\BinaryOperatorsTestBaseline2.txt" />
     <Content Include="Semantics\BinaryOperatorsTestBaseline3.txt" />
     <Content Include="Semantics\BinaryOperatorsTestBaseline4.txt" />
     <Content Include="Semantics\BinaryOperatorsTestBaseline5.txt" />
-    <Content Include="Semantics\BinaryOperatorsTestSource1.vb" />
-    <Content Include="Semantics\BinaryOperatorsTestSource2.vb" />
-    <Content Include="Semantics\BinaryOperatorsTestSource3.vb" />
-    <Content Include="Semantics\BinaryOperatorsTestSource4.vb" />
-    <Content Include="Semantics\BinaryOperatorsTestSource5.vb" />
-    <Content Include="Semantics\OverloadResolutionTestSource.vb" />
-    <Content Include="Semantics\PrintResultTestSource.vb" />
     <EmbeddedResource Include="Binding\T_1247520.cs" />
     <EmbeddedResource Include="Binding\T_68086.vb" />
-    <EmbeddedResource Include="Resource.resx">
+    <Compile Remove="Binding\T_68086.vb" />
```

Comment:
Yep that would be nice - if it's .either EmgbeddedResource or Content, it probably should get automatic removes. I'll log a SDK bug.